### ISSUE TITLE
WCAG: Dropdown/select/combobox do not read out label (VoiceOver/Chrome)

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#1667]
### Description of the bug

I cant get the select component to read the label when focus enters the component.
![Skjermbilde 2023-11-29 kl  10 44 43](https://github.com/Altinn/app-frontend-react/assets/33222679/fd0d2339-1e1a-41b7-9364-2b376713a988)

**NB:** This is probably an issue solved by the designsystem component. But needs to be retested when it is implemented here. In cases where we dont need to style the list or have search / multiselect options we could switch to the native select. 

- [ ] Retest using NVDA/JAWS and other browsers

### Steps To Reproduce

1. Go to an app with a select
2. Use a screen reader and see what information it relays when focus is on the component

**Comment by bjosttveit** [comment]
@Febakke I am not sure I understand the issue completely, from the screenshot it looks like it reads the label when opening the dropdown, is the issue that it did not read the label when focusing on it _before_  opening it up?

**Comment by Febakke** [comment]
@bjosttveit 
Correct, as with other form inputs the label and description (if present) should be read when the input gets focus. 

Seems like there has been some changes to your combobox since this was reported, might be fixed by that?

**Comment by bjosttveit** [comment]
Yeah we are using the new `Combobox` now, tested on the language selector, and it does read the label upon focus now

**Comment by bjosttveit** [comment]
Solved by https://github.com/Altinn/app-frontend-react/pull/2140

**Comment by bjosttveit** [comment]
Solved by https://github.com/Altinn/app-frontend-react/pull/2140
